src/components/Pagination/Pagination.tsx

```diff
@@ -32,6 +32,7 @@ export type PaginationProps = {
   paginationStyleItemActive?: ViewStyle;
   paginationStyleItemInactive?: ViewStyle;
   onPaginationSelectedIndex?: () => void;
+  disablePaginationPress?: boolean;
```

Comment:
Could you rename this to paginationIsDisabled

Reply:
I can, but I think `paginationIsDisabled` implies that there will be no pagination. 

What is it about this name that you don't like?

Would you consider `paginationTapDisabled` or `paginationInteractionsDisabled`?

Reply:
Both paginationTapDisabled or paginationInteractionsDisabled are good for me. I want to keep the idea of using a prefix, eg `paginationXXXX`

Reply:
can we merge it?

Reply:
Hi @gusgard , would you consider merging this if I submit a PR with the name `paginationTapDisabled`?

Reply:
See https://github.com/gusgard/react-native-swiper-flatlist/pull/129

Reply:
Yes, I will merge it in a bit

Reply:
Released with https://github.com/gusgard/react-native-swiper-flatlist/releases/tag/3.0.16